src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -721,6 +721,182 @@ RppStatus rppt_resize_crop_mirror_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    RppLayoutParams srcLayoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+
+    if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::U8))
+    {
+        RpptImagePatchPtr internalDstImgSizes = (RpptImagePatch *) calloc(dstDescPtr->n, sizeof(RpptImagePatch));
```

Comment:
There seems to be common code for all bitdepths L738 to L746
Please move this code to outside the if statement at L735